### ISSUE TITLE
Let user chose between Python 2 or 3 during installation

### ISSUE THREAD

**Issue by nimiq** [nimiq/promptastic#19]
During installation ask the user if she wants to use Python 2 or 3.
The installation will automatically change the first line in *promptastic.py*:
`#!/usr/bin/env python3`

**Comment by dzogrim** [comment]
Yep, I agree with that! Could be very nice to get the choice for Python3 that handle more than ascii chars in the prompt… a lot of failure with Python2 BTW. 👍 
Thanks!
